earthpy/io.py

```diff
@@ -91,6 +93,9 @@
 DATA_NAME = op.join("earth-analytics", "data")
 ALLOWED_FILE_TYPES = ["file", "tar", "tar.gz", "zip"]
 
+# Create logger object
+logger = logging.getLogger("io")
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```

Best practice is to use `__name__`.

Reply:
thank you @sgillies !!! accepted this suggestion